plugins/techdocs-addons-contrib/src/ReportIssue/ReportIssue.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useEffect } from 'react';
+
+import { makeStyles, Portal, Paper } from '@material-ui/core';
+
+import { useGitTemplate, useGitRepository } from './hooks';
+import { ReportIssueTemplateBuilder } from './types';
+import {
+  PAGE_MAIN_CONTENT_SELECTOR,
+  PAGE_FEEDBACK_LINK_SELECTOR,
+  ADDON_FEEDBACK_CONTAINER_ID,
+  ADDON_FEEDBACK_CONTAINER_SELECTOR,
+} from './constants';
+import { IssueLink } from './IssueLink';
+
+import {
+  useShadowRootElements,
+  useShadowRootSelection,
+} from '@backstage/plugin-techdocs-react';
```

Comment:
We might need to double check if Jest mocks will correctly hook here, otherwise we can fallback to relative imports for now until we found a way to ensure the mocks are applied properly.

Which means we will take care of that in this PR: https://github.com/backstage/backstage/pull/10713